templates/track_process.erb

```diff
@@ -0,0 +1,15 @@
+vrrp_track_process <%= @name %> {
```

Comment:
epp template is welcome.

Reply:
@Dan33l is right that this is often preferred but I'm also fine with `erb` (especially given all the existing templates are erb).